src/Sentry/DefaultSentryScopeStateProcessor.cs

```diff
@@ -38,7 +38,7 @@ public void Apply(Scope scope, object state)
 
                     break;
                 }
-#if HAS_VALUE_TUPLE
+#if !NET461
```

Comment:
Note: we can bring `ValueTuple` to .NET461 with: https://www.nuget.org/packages/System.ValueTuple/
This will remove the need for this if/defs

Reply:
I had that originally and it cause a WORLD of pain (binding redirects), so I removed it